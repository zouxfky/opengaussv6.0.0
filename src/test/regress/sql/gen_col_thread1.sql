begin;
update gtest1 set a=a||'thread1';
select pg_sleep(7);
commit;
