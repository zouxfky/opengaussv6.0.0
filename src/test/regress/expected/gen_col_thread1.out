begin;
update gtest1 set a=a||'thread1';
select pg_sleep(7);
 pg_sleep 
----------
 
(1 row)

commit;
